index.html

```diff
@@ -0,0 +1,150 @@
+                                        Hex to Text
+                                    </button>
+                                </h5>
+                            </div>
+
+                            <div id="HexToText" class="collapse" aria-labelledby="headingOne"
+                                 aria-expanded="true">
+                                <div class="card-body">
+                                    <div class="form-group green-border-focus">
+                                        <label for="hexInput">Hex Value:</label>
+                                        <textarea class="form-control rounded-0" rows="5" id="hexInput" value=""
+                                            placeholder="Hex input..."></textarea>
+                                            <p style="color: red" id="errorMsg"></p>
+                                        <div class="row d-flex align-items-center mb-3 mt-4">
+
+
+                                            <div class="col-md-12">
+                                                <div class="text-left">
+                                                    <button type="button"
+                                                        onclick="convertHex(document.getElementById('hexInput').value)"
+                                                        class="btn btn-outline-success btn-rounded z-depth-1a">Convert</button>
+                                                </div>
+                                            </div>
+
+
+                                        </div>
+
+                                        <label for="textOutput">Text Output:</label>
+                                        <textarea class="form-control rounded-0" rows="5" id="textOutput" value=""
+                                            disabled placeholder="Text Output..."></textarea>
+                                            
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="card">
+                            <div class="card-header" id="headingTwo">
+                                <h5 class="mb-0">
+                                    <button class="btn btn-link collapsed" data-toggle="collapse"
+                                        data-target="#TextToHex" aria-expanded="false" aria-controls="collapseTwo">
+                                        Text to Hex
+                                    </button>
+                                </h5>
+                            </div>
+                            <div id="TextToHex" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
+                                <div class="card-body">
+                                    <div class="form-group green-border-focus">
+                                        <label for="hexInput">Hex Value:</label>
```

Comment:
```suggestion
                                        <label for="hexInput">Text Value:</label>
```
Simple copy pasta mistake I am sure

Reply:
Yes, you are correct :-))) I noticed after PR that I forgot to changed the labels. 
Thank you!